packages.json

```diff
@@ -8593,5 +8593,16 @@
     "description": "Text editor to speed up testing code snippets",
     "license": "MIT",
     "web": "https://github.com/genotrance/snip"
+  },
+  {
+    "name": "moduleinit",
+    "url": "https://github.com/skunkiferous/moduleinit",
+    "method": "git",
+    "tags": [
+      "v0.1.0"
```

Comment:
This isn't where you put git tags, Nimble can read those independently from your Git repo.

The tags are just categories, see other package's tags.